### PR TITLE
Applying MEQP1 Standard with phpcbf command

### PR DESCRIPTION
**Pull request by roberto-ebizmarts**
Applying MEQP1 Standard with phpcbf command

### CODE REVIEW

File: app/code/community/Ebizmarts/MailChimp/Helper/Data.php [hunk 1]
```diff
@@ -4206,10 +4217,10 @@ public function getPromoConfig($scopeId, $scope = null)
     public function getSyncFlagDataHtml($dat, $string)
     {
         $syncFlagDataArray = $this->getSyncFlagDataArray($dat);
-        if ($syncFlagDataArray[Ebizmarts_MailChimp_Model_System_Config_Source_Account::SYNC_FLAG_STATUS]
-            != Ebizmarts_MailChimp_Model_System_Config_Source_Account::IN_PROGRESS) {
-            if ($syncFlagDataArray[Ebizmarts_MailChimp_Model_System_Config_Source_Account::SYNC_FLAG_STATUS]
-                == Ebizmarts_MailChimp_Model_System_Config_Source_Account::FINISHED) {
+        if ($syncFlagDataArray[Ebizmarts_MailChimp_Model_System_Config_Source_Account::SYNC_FLAG_STATUS]!= Ebizmarts_MailChimp_Model_System_Config_Source_Account::IN_PROGRESS
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Helper/Data.php]: Line exceeds 120 characters.

File: app/code/community/Ebizmarts/MailChimp/Helper/Data.php [hunk 2]
```diff
@@ -4206,10 +4217,10 @@ public function getPromoConfig($scopeId, $scope = null)
...
+        ) {
+            if ($syncFlagDataArray[Ebizmarts_MailChimp_Model_System_Config_Source_Account::SYNC_FLAG_STATUS]== Ebizmarts_MailChimp_Model_System_Config_Source_Account::FINISHED
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Helper/Data.php]: Line exceeds 120 characters.

File: app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php [hunk 3]
```diff
@@ -184,25 +184,25 @@ protected function _unsubscribe(array $data)
...
-                        break;
+                case 'delete':
+                    //if config setting "Webhooks Delete action" is set as "Delete customer account"
+                    if (Mage::getStoreConfig(
+                        Ebizmarts_MailChimp_Model_Config::GENERAL_UNSUBSCRIBE,
+                        $subscriber->getStoreId()
+                    )
+                    ) {
+                        $subscriber->delete();
+                    } elseif ($subscriber->getSubscriberStatus() !=                        Mage_Newsletter_Model_Subscriber::STATUS_UNSUBSCRIBED
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php]: Line exceeds 120 characters.

File: app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php [hunk 4]
```diff
@@ -184,25 +184,25 @@ protected function _unsubscribe(array $data)
...
+                    ) {
+                        $helper->unsubscribeMember($subscriber);
+                    }
+                    break;
+                case 'unsub':
+                    if ($subscriber->getSubscriberStatus() !=                        Mage_Newsletter_Model_Subscriber::STATUS_UNSUBSCRIBED
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php]: Line exceeds 120 characters.

File: app/code/community/Ebizmarts/MailChimp/Model/System/Config/Source/Store.php [hunk 5]
```diff
@@ -23,7 +23,8 @@ class Ebizmarts_MailChimp_Model_System_Config_Source_Store
 
     /**
      * Ebizmarts_MailChimp_Model_System_Config_Source_Store constructor.
-     * @param $params
+     *
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Model/System/Config/Source/Store.php]: Why extra line?

**Review by SantiParagarino — CHANGES_REQUESTED** [review]

File: app/code/community/Ebizmarts/MailChimp/Model/Api/Batches.php [hunk 6]
```diff
@@ -828,8 +830,8 @@ protected function _getError($type, $mailchimpStoreId, $id, $response)
         if ($type == Ebizmarts_MailChimp_Model_Config::IS_PRODUCT) {
             $dataProduct = $this->getDataProduct($helper, $mailchimpStoreId, $id, $type);
             if ($dataProduct->getMailchimpSyncDeleted()
-                || $dataProduct->getMailchimpSyncError() ==
-                Ebizmarts_MailChimp_Model_Api_Products::PRODUCT_DISABLED_IN_MAGENTO) {
+                || $dataProduct->getMailchimpSyncError() ==Ebizmarts_MailChimp_Model_Api_Products::PRODUCT_DISABLED_IN_MAGENTO
```
[the remark this answers is not in the document]
Comment (roberto-ebizmarts) [reply app/code/community/Ebizmarts/MailChimp/Model/Api/Batches.php]: Solved with a variable saving Ebizmarts_MailChimp_Model_Api_Products::PRODUCT_DISABLED_IN_MAGENTO

File: app/code/community/Ebizmarts/MailChimp/Helper/Data.php (hunk 2 above)
Comment (roberto-ebizmarts) [reply app/code/community/Ebizmarts/MailChimp/Helper/Data.php]: Solved

File: app/code/community/Ebizmarts/MailChimp/Model/Api/Batches.php [hunk 7]
```diff
@@ -1051,8 +1053,8 @@ protected function setItemAsModified($helper, $mailchimpStoreId, $id, $type)
             $isMarkedAsDeleted = $dataProduct->getMailchimpSyncDeleted();
 
             if (!$isMarkedAsDeleted
-                || $dataProduct->getMailchimpSyncError() !=
-                    Ebizmarts_MailChimp_Model_Api_Products::PRODUCT_DISABLED_IN_MAGENTO) {
+                || $dataProduct->getMailchimpSyncError() !=                Ebizmarts_MailChimp_Model_Api_Products::PRODUCT_DISABLED_IN_MAGENTO
```
[the remark this answers is not in the document]
Comment (roberto-ebizmarts) [reply app/code/community/Ebizmarts/MailChimp/Model/Api/Batches.php]: Solved with variable created saving: Ebizmarts_MailChimp_Model_Api_Products::PRODUCT_DISABLED_IN_MAGENTO

File: app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php [hunk 8]
```diff
@@ -184,25 +184,25 @@ protected function _unsubscribe(array $data)
...
+                    )
+                    ) {
+                        $subscriber->delete();
+                    }
+                    elseif ($subscriber->getSubscriberStatus() != $statusUnsubscribed) {
+                        $helper->unsubscribeMember($subscriber);
+                    }
+                    break;
+                case 'unsub':
+                    if ($subscriber->getSubscriberStatus() != Mage_Newsletter_Model_Subscriber::STATUS_UNSUBSCRIBED
```
Comment (Santiagoebizmarts) [inline app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php]: Reuse same variable $statusUnsubscribed as above.

File: app/code/community/Ebizmarts/MailChimp/Helper/Data.php (hunk 1 above)
Comment (roberto-ebizmarts) [reply app/code/community/Ebizmarts/MailChimp/Helper/Data.php]: Solved:
$syncFlagStatus = Ebizmarts_MailChimp_Model_System_Config_Source_Account::SYNC_FLAG_STATUS;
        $inProgress = Ebizmarts_MailChimp_Model_System_Config_Source_Account::IN_PROGRESS

**Review by SantiParagarino — CHANGES_REQUESTED** [review]

File: app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php (hunk 3 above)
Comment (roberto-ebizmarts) [reply app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php]: Solved:
$statusUnsubscribed = Mage_Newsletter_Model_Subscriber::STATUS_UNSUBSCRIBED;

File: app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php (hunk 4 above)
Comment (roberto-ebizmarts) [reply app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php]: Solved

File: app/code/community/Ebizmarts/MailChimp/Model/System/Config/Source/Store.php (hunk 5 above)
Comment (roberto-ebizmarts) [reply app/code/community/Ebizmarts/MailChimp/Model/System/Config/Source/Store.php]: Solved

File: app/code/community/Ebizmarts/MailChimp/Model/Api/Carts.php [hunk 9]
```diff
@@ -27,7 +27,9 @@ class Ebizmarts_MailChimp_Model_Api_Carts
      */
     public function createBatchJson($mailchimpStoreId, $magentoStoreId)
     {
-        /** @var Ebizmarts_MailChimp_Helper_Data $helper */
+        /**
```
[the remark this answers is not in the document]
Comment (roberto-ebizmarts) [reply app/code/community/Ebizmarts/MailChimp/Model/Api/Carts.php]: Solved

File: app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php (hunk 8 above)
Comment (roberto-ebizmarts) [reply app/code/community/Ebizmarts/MailChimp/Model/ProcessWebhook.php]: Done.

**Review by SantiParagarino — APPROVED** [review]

**Pull request merged**